static/js/hotkey.js

```diff
@@ -574,6 +581,16 @@ exports.process_hotkey = function (e, hotkey) {
         case 'stream_cycle_forward':
             navigate.cycle_stream('forward');
             return true;
+        case 'keyboard_sub':
+            if (exports.is_subs()) {
+                subs.keyboard_sub();
+                return true;
```

Comment:
make sure to put the `return true` outside of the if statement so the case block will break.